build.sbt

```diff
@@ -61,7 +61,6 @@ lazy val core = project
       "prepareCi",
       "scalafmtAll;scalafmtSbt;scalafixAll;+ test;docs/tlSite;mimaReportBinaryIssues"
     ),
-    mimaFailOnProblem := false, // do not fail on MiMa failures until library reaches 1.0.0
```

Comment:
I pushed this change, as I have realized this is not necessary. Whether to do MiMa checks or not is supposed to be controlled via the `tlBaseVersion` setting.